src/test/java/com/volunteer/enterprise/EnterpriseApplicationTests.java

```diff
@@ -1,13 +1,27 @@
 package com.volunteer.enterprise;
 
+import com.volunteer.enterprise.dto.Volunteer;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 @SpringBootTest
 class EnterpriseApplicationTests {
 
     @Test
-    void contextLoads() {
+    void confirmJonSnow_outputsJonSnow() {
+        Volunteer volunteer = new Volunteer();
+        volunteer.setVolunteerID(86452);
+        volunteer.setVolunteerEmail("Drogon145@gmail.com");
+        volunteer.setVolunteerUser("Jon Snow");
+        volunteer.setVolunteerPass("Passw0rd@01");
+        assertEquals(86452, volunteer.getVolunteerID());
+        assertEquals("Drogon145@gmail.com", volunteer.getVolunteerEmail());
+        assertEquals("Jon Snow", volunteer.getVolunteerUser());
+        assertEquals("Passw0rd@01", volunteer.getVolunteerPass());
+
```

Comment:
Added test for this project to compile. Changed the name of the method to a more sensible name.